configs/next/packages/tbb/sources

```diff
@@ -56,16 +56,3 @@ atsrc_package_verify_make_check_log ()
 		return ${?}
 	fi
 }
-
-atsrc_get_patches ()
-{
-	at_get_patch \
-		https://github.com/powertechpreview/powertechpreview/raw/42cfbcf3d4a2fb7f9216069397df0dff9fc0f651/TBB%20PowerPC%20Patches/4.3/ppc64-tbb-4.3-fix.tgz \
-		11708da1a42542608a345c8f0b71dfc6 || return ${?}
-}
-
-atsrc_apply_patches ()
-{
-	tar xzf ppc64-tbb-4.3-fix.tgz || return ${?}
-	patch -p1 < inteltbb_set_compiler.patch || return ${?}
```

Comment:
OK.

Upstream included this change as can be seen in this [link](https://github.com/01org/tbb/commit/4cebdd9a1c9f0c56c4ddcbbb400447ab2a11fad4#diff-76933eb251f8863acd03337b951b04fb).